seed/csharp-sdk/exhaustive/explicit-namespaces/src/SeedExhaustive/NoAuth/NoAuthClient.cs

```diff
@@ -0,0 +1,38 @@
+using System.Net.Http;
+using SeedExhaustive.Core;
+
+#nullable enable
+
+namespace SeedExhaustive.NoAuth;
```

Comment:
example of diff here
previously would've been just 
`namespace SeedExhaustive;`